src/components/bulkConcepts/component/CardBody.jsx

```diff
@@ -10,7 +10,7 @@ const CardBody = ({ title, body }) => (
 
 CardBody.propTypes = {
   title: PropTypes.string.isRequired,
-  body: PropTypes.string.isRequired,
+  body: PropTypes.any.isRequired,
```

Comment:
Is this also about testing the untested areas?

Reply:
Yes, this is part of the first bullet point that handles restructuring of the code in order for it to be tested. After testing, I got warnings on the terminal console and solved it as it would get different fields depending on the data in the body.

Reply:
The restructuring should happen in a separate ticket and pull request.

Reply:
Let the restructuring have its own ticket and pull request. The warnings will all go away once the various tickets are created, reviewed, and merged.